caffe2/core/test_utils.h

```diff
@@ -140,6 +144,9 @@ class NetMutator {
     return *this;
   }
 
+  // Set device name for the last created op.
+  NetMutator& setDeviceOptionName(const string& name);
```

Comment:
Is `string` from `std`? It looks a bit inconsistent that in some places we have `std::string` and in others just `string`.